src/redux/reducer/userSlice.jsx

```diff
@@ -0,0 +1,112 @@
+import { createSlice } from '@reduxjs/toolkit';
+import {
+  fetchLoginThunk,
+  addCorrectionRequestThunk,
+  deleteCorrectionRequestThunk,
+  addTaskThunk,
+  deleteTaskThunk,
+  updateTaskThunk,
+} from './userThunks';
+
+let initialState = {
+  data: {},
+  status: '',
+  isLogin: false,
+  error: null,
+  showSuccessBox: false,
+};
+
+const userSlice = createSlice({
+  name: 'user',
+  initialState,
+  reducers: {
+    clearUser: (state) => {
+      state.data = {};
+      state.status = '';
+      state.isLogin = false;
+      state.error = null;
+      state.showSuccessBox = false;
+    },
```

Comment:
```suggestion
  clearUser: (state) => {
      return initialState;
    },
```

초기 상태를 사용해 이런식으로 작성할 수 있을까요?